<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerNamesPage</class>
 <widget class="QWidget" name="PlayerNamesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="player1groupBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>300</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>Player 1: X</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="player1LineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="homeBottom">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startGameButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>510</y>
     <width>100</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Game!</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="player2groupBox_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>300</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>Player 2: O </string>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="1">
     <widget class="QLineEdit" name="player2LineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
